rero_ils/modules/vendors/api.py

```diff
@@ -72,11 +72,13 @@ class Vendor(IlsRecord):
     def order_email(self):
         """Shortcut for vendor order email.
 
-        if the order email does not exist, it returns the default contact email
+        :return the best possible email to use for this vendor. If the specific
+                order contact information does not exist, the default contact
+                information will be used.
         """
-        return self.get(
-            'order_contact', self.get('default_contact', {})
-        ).get('email')
+        return self\
+            .get('order_contact', self.get('default_contact', {}))\
+            .get('email')
```

Comment:
in this case, we need to validate that each vendor has at least one email. I did not find this check in the vendor class

Reply:
Not really :-) 
First this will be check about an `order_contact` or an `default_contact` dict attribute. If none are found, we will use an empty dict as default value.
On this dict, we get the `email` attribute. If this attribute isn't present, then `None` is returned.
So, for me, this method is correct.

But, in the UI, I need to check if vendor has an email (order or default). If not, then we should not select 'send to vendor' option.

Reply:
clear now, thanks.
Does it mean the `send_order` will fail if we use the `console` instead of the UI for vendors without email?